### PR TITLE
Workstation Release management: Include mention of dbgsym package in apt-prod instructions 

### PR DESCRIPTION
**Pull request by rocodes**
## Status

Ready for review

## Description of Changes

* Description:  Add note about dbgsym package in workstation release section

## Checklist (Optional)

- [x] Doc linting (`make docs-lint`) passed locally
- [ ] Doc link linting (`make docs-linkcheck`) passed
- [x] You have previewed (`make docs`) docs at http://localhost:8000

### CODE REVIEW

File: docs/workstation_release_management.rst [hunk 1]
```diff
@@ -88,7 +88,7 @@ Step 5: Build and deploy the packages to ``apt-qa``
    make build-debs
 
 3. Save and publish :doc:`build metadata <build_metadata>`.
-4. Add your packages to a new branch called ``release`` in https://github.com/freedomofpress/securedrop-apt-prod.
+4. Add your packages to a new branch called ``release`` in https://github.com/freedomofpress/securedrop-apt-prod. For Packages with compiled Rust (currently ``securedrop-proxy``), also include the ``-dbgsym`` package.
```
Comment (legoktm) [inline docs/workstation_release_management.rst:91]: It's anything that has compiled code, which is proxy and once https://github.com/freedomofpress/securedrop-client/pull/2178 is merged, also the client, which has a few Python extensions that are compiled.

Comment (legoktm) [reply docs/workstation_release_management.rst:91]: Maybe we can just say something like "Include all packages built by the client, including ``-dbgsym`` ones."?

**Comment by rocodes** [comment]
thank you, addressed in https://github.com/freedomofpress/securedrop-dev-docs/pull/184/commits/20454b97dfee408c3a1cee04af8a5e15ae44d184

**Review by legoktm — APPROVED** [review]
Thanks!

**Pull request merged**